packages/server/tests/acceptance/index.spec.ts

```diff
@@ -90,9 +105,47 @@ describe('RPC Server Acceptance Tests', function () {
 
     // cache start balance
     startOperatorBalance = await global.servicesNode.getOperatorBalance();
+    const initialAccount = await global.servicesNode.createInitialAliasAccount(
+      RELAY_URL,
+      CHAIN_ID,
+      Utils.generateRequestId(),
+    );
+
+    global.accounts = new Array<AliasAccount>(initialAccount);
+    // wait 3 seconds to propagate to mirror node all needed changes.
+    await new Promise((r) => setTimeout(r, 3000));
```

Comment:
Q: Why wait instead of poll.
Waiting can be problematic and hides failure reasons

Reply:
Yep, you're right. Changed.